libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/MapboxNativeNavigatorImpl.kt

```diff
@@ -245,8 +245,8 @@ object MapboxNativeNavigatorImpl : MapboxNativeNavigator {
      */
     override suspend fun getRoute(url: String): Expected<RouterError, String> {
         return suspendCoroutine { continuation ->
-            nativeRouter!!.getRoute(url) {
-                continuation.resume(it)
+            nativeRouter!!.getRoute(url) { expected, _ ->
```

Comment:
Noting that `Router` is currently `@Deprecated` 👀 

```java
/** @deprecated Public Router class for onboard router is redundant. Please use: {@link com.mapbox.navigator.RouterFactory} instead. */
```

Also the `_` is the `RouterOrigin`.

Reply:
I think it's our next step and `Router` already deprecated, I left todo on top of the class. Also, we don't use `RouterOrigin`, so it's marked as `_`